docker_hub/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:latest
+ARG ADAPT_VERSION
+RUN apk --update add --no-cache nodejs npm openssh \
+  && rm -rf /var/cache/apk/*
+RUN npm install -g @adpt/cli@$ADAPT_VERSION
+RUN mkdir /src
+VOLUME ["/src"]
```

Comment:
Although you have it in the README, I'd suggest adding a volume for `/root/.local/share/adapt` as a best practice for telling users they should consider what they might want to mount there.

Reply:
Done.